pages/compiling/installation-reqs.rst

```diff
@@ -70,62 +69,94 @@ Conda Requirements
+     # - libgomp  # optional (OpenMP implementation when using GCC)
+     - cython  # needed to build python package
+     - numpy  # needed to build python package
+     - pytest  # needed for the Python test suite
+     - ruamel.yaml  # needed for converter scripts
+     # - sphinx  # optional (used for documentation)
+     # - doxygen  # optional (used for documentation)
+     # - graphviz  # optional (needed for documentation)
+     # - python-graphviz  # optional (needed for reaction path diagrams)
+     # - h5py  # optional (needed for HDF/H5 output)
+     # - pandas  # optional (needed for pandas interface)
+     # - scipy  # optional (needed for some examples)
+     # - matplotlib  # optional (needed for plots)
+     - ipython  # optional (needed for nicer interactive command line)
+     # - jupyter  # optional (needed for Jupyter Notebook)
+
+  The environment is then created and activated using
 
   .. code:: bash
 
-     conda create --name cantera python=3 scons cython boost numpy ruamel.yaml
-     conda activate cantera
+     conda env create -f environment.yaml
+     conda activate ct-build
 
-* (Optional) If you also want to build the documentation, after you've created the environment and
-  activated it, you'll also need to install the following dependencies
+  After creating the enviroment, it can be updated from within ``ct-build`` using
 
   .. code:: bash
 
-     conda install sphinx doxygen graphviz
-     pip install sphinxcontrib-matlabdomain sphinxcontrib-katex sphinxcontrib-doxylink
-
-* After you've :ref:`cloned the source code <sec-source-code>`, configure the Cantera build by
-  adding the following options to a file called ``cantera.conf`` in the root of the source directory
-  (creating the file if it doesn't exist).
-
-  On macOS and Linux, add the following code to your ``cantera.conf`` file:
-
-  .. code:: python
+     conda env update -f environment.yaml --prune
 
-     python_package = 'full'
-     boost_inc_dir = '/path/to/conda/install/folder/envs/cantera/include'
+* (Optional) If you want to override external libraries packaged with Cantera
+  (``sundials``, ``fmt``, ``eigen``, ``yaml-cpp``), simply uncomment corresponding
+  lines in the file ``environment.yaml`` above. Note that specific versions can be
+  forced by providing version numbers (example: replace ``sundials`` by
+  ``sundials=5.8`` to install version ``5.8``).
 
-  On Windows, add the following code to your ``cantera.conf`` file:
+* (Optional) If you want to build the documentation, make sure to have uncommented
+  lines containing ``sphinx``, ``doxygen`` and ``graphviz`` in the file
+  ``environment.yaml`` above. In addition, you also need to install the following
+  dependencies
 
-  .. code:: python
+  .. code:: bash
 
-     python_package = 'full'
-     boost_inc_dir = '/path/to/conda/install/folder/envs/cantera/Library/include'
+     pip install sphinxcontrib-matlabdomain sphinxcontrib-katex sphinxcontrib-doxylink
```

Comment:
These can be listed in `environment.yaml`:

```yaml
     - ipython  # optional (needed for nicer interactive command line)
     # - jupyter  # optional (needed for Jupyter Notebook)
     - pip:
       - sphinxcontrib-matlabdomain
       - ...
```

Reply:
Neat. Didn't know that!